AK/Assertions.cpp

```diff
@@ -89,7 +89,7 @@ extern "C" {
 
 void ak_verification_failed(char const* message)
 {
-    ERRORLN("VERIFICATION FAILED: {}", message);
+    ERRORLN("\033[31;1mVERIFICATION FAILED\033[0m: {}", message);
```

Comment:
should we check isatty here? though I guess that's not quite correct because on android these go to `__android_log_write()`

Reply:
Checking `isatty` could become quite some #ifdef soup here, unless someone knows offhand what fd `bochs_debug_output` goes to on x86, or whatever aarch64 writes to.

Reply:
hmm, well I guess we could just do:
```c++
#if defined(AK_OS_SERENITY) || defined(AK_OS_ANDROID)
bool colorize_output = true;
#else
bool colorize_output = isatty(STDERR_FILENO) == 1;
#endif

if (colorize_output)
    ERRORLN("\033[31;1mVERIFICATION FAILED\033[0m: {}", message);
else
    ERRORLN("VERIFICATION FAILED: {}", message);
```